analyzer/src/main/kotlin/managers/Gradle.kt

```diff
@@ -140,7 +140,7 @@ class Gradle(
 
             log.info {
                 "Will apply the following system properties defined in file '$gradlePropertiesFile': " +
-                        gradleSystemProperties
+                        gradleSystemProperties.map { "${it.first} = ${it.second}" }
```

Comment:
Or use `joinToString(separator = "\n        ") { "${it.first} = ${it.second}" }` to print one property per line.

Reply:
Done.